modules/apps/frontend-js/frontend-js-a11y-web/package.json

```diff
@@ -1,6 +1,5 @@
 {
 	"dependencies": {
-		"@liferay/frontend-js-react-web": "*",
```

Comment:
Hey @natocesarrego Looks like the SF failure is related to this change.

Reply:
Hey @ling-alan-huang, we need to keep this dependency because it's importing a TypeScript resource from another module, the `frontend-js-react-web`, therefore we should include the module `frontend-js-a11y-web` as an exception too.